cypress/integration/transforms_spec.js

```diff
@@ -15,243 +9,236 @@ import sampleTransform from "../fixtures/sample_transform";
 const TRANSFORM_ID = "test_transform_id";
 
 describe("Transforms", () => {
-    before(() => {
-      // Delete all indices
-      cy.deleteAllIndices();
-
-      // Load ecommerce data
-      cy.request({
-        method: 'POST',
-        url:`${Cypress.env("opensearch_dashboards")}/api/sample_data/ecommerce`,
-        headers: {
-          'osd-xsrf': true
-        }
-      }).then((response) => {
-        expect(response.status).equal(200);
-      });
+  before(() => {
```

Comment:
Are all these lines showed as modified due formatting the leading spaces in each line?

Reply:
yes